src/components/core/button/toggle-button.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+const ToggleButton = ({ isActive, onClick, label, ariaLabel }) => {
```

Comment:
isRequired for propTypes is useful, but it's not mandatory. Could we set a default value for each prop in this core component to help prevent nullish value issues?